compute/instances.go

```diff
@@ -103,6 +104,38 @@ func (gmi *GetInstanceInput) Validate() error {
 	return nil
 }
 
+func (c *InstancesClient) Count(ctx context.Context, input *ListInstancesInput) (int, error) {
```

Comment:
Here, the `ListInstancesInput` might be an interesting use-case for much dreaded type aliases added in Go 1.9. So that would be:

```go
type CountInstancesInput = ListInstancesInput
```

See https://golang.org/doc/go1.9#language for more details.

Reply:
Example: https://play.golang.org/p/ftoqWLjx154